library.json

```diff
@@ -31,7 +31,10 @@
     {
       "name": "Bounce2",
       "version": "^2.1.0"
-    }
+    },
+	{
+	  "name": "ESPAsyncWebServer"
```

Comment:
Can you add a "version" field with the `^`, so that if there's a breaking change it still works.

Reply:
Problem, they dont have a version [https://github.com/me-no-dev/ESPAsyncWebServer](https://github.com/me-no-dev/ESPAsyncWebServer). Shall i just add a commit hash ?

Reply:
OK, nevermind then. 😉